pinot-hadoop-filesystem/src/main/java/com/linkedin/pinot/filesystem/AzurePinotFS.java

```diff
@@ -88,8 +91,15 @@ public boolean move(URI srcUri, URI dstUri) throws IOException {
 
   @Override
   public boolean copy(URI srcUri, URI dstUri) throws IOException {
-    // TODO: Add this method. Not needed for now because file will be in final location
-    throw new UnsupportedOperationException("Cannot copy from AzureFS to AzureFS");
+    if (exists(dstUri)) {
```

Comment:
Should we have overwrite as parameter passed? So we don't always overwrite.

Reply:
I think we don't have that use case in Pinot and I don't anticipate us needing that as these are used for segment upload, refreshing segments on server side, etc.

Reply:
When we designed, other filesystems default overwrite, so I wanted to keep it consistent.